src/tests/types.rs

```diff
@@ -0,0 +1,8 @@
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+pub struct AxumErrorResponse {
+    pub happened_at: String,
+    pub message: String,
+    pub resource: String,
+}
```

Comment:
isn't this part of `axum` crate already ?

Reply:
Actually we manually specify that here - https://github.com/Astraly-Labs/indexer-service/blob/490fdb62e43dd3391a82b18037dea57f65793586/src/domain/models/indexer.rs#L93-L93. I have changed this to a struct now and used that struct in both places